build-ipa.sh

```diff
@@ -23,11 +23,11 @@ mv /tmp/packages/*.rpm .
 rm var/lib/rpm/.rpm.lock
 
 chroot . /bin/bash -x <<'EOF'
-rpm -Uvh \
-openstack-ironic-python-agent*.rpm \
-python3-bcrypt*.rpm \
-python3-ironic-python-agent*.rpm \
-python3-ironic-lib*.rpm || { exit 1; }
+dnf install -y \
+./openstack-ironic-python-agent*.rpm \
```

Comment:
Do these glob patterns need to appear in single quotes to avoid shell expansion?

Reply:
probably it's not necessary in this case considering that we know the rpms in advance
I'm actually thinking to move the logic entirely here from the Dockerfile as it's very confusing at the moment
In the end I hope will be able to avoid doing this and just installing normally using dnf

Reply:
I just changed to install with dnf by default so this is resolved, in a way